services/src/projects/project.rs

```diff
@@ -78,14 +72,16 @@ impl Project {
     }
 }
 
-#[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
+#[derive(Debug, PartialEq, Serialize, Deserialize, Clone, ToSql, FromSql)]
 pub struct STRectangle {
+    pub spatial_reference: SpatialReference,
```

Comment:
Maybe `SpatialReferenceOption` to allow unreferenced rectangles

Reply:
we have `SpatialReference::Unreferenced` now

Reply:
Yes, but this is in your branch still `SpatialReferenceOption::None`.